app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -21,37 +25,206 @@ import dagger.Component
 import dagger.Module
 import dagger.Provides
 import kotlinx.coroutines.CoroutineDispatcher
+import org.hamcrest.Matchers.not
+import org.junit.Before
 import org.junit.Ignore
 import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.oppia.app.R
+import org.oppia.app.testing.ExplorationInjectionActivity
+import org.oppia.domain.exploration.ExplorationDataController
 import org.oppia.domain.exploration.TEST_EXPLORATION_ID_30
+import org.oppia.domain.topic.FRACTIONS_EXPLORATION_ID_0
+import org.oppia.domain.topic.RATIOS_EXPLORATION_ID_0
+import org.oppia.util.networking.NetworkConnectionUtil
 import org.oppia.util.threading.BackgroundDispatcher
 import org.oppia.util.threading.BlockingDispatcher
+import javax.inject.Inject
 import javax.inject.Singleton
 
 /** Tests for [ExplorationActivity]. */
 @RunWith(AndroidJUnit4::class)
 class ExplorationActivityTest {
```

Comment:
<img width="1638" alt="Screenshot 2019-11-29 at 12 02 46 AM" src="https://user-images.githubusercontent.com/9396084/69827009-9cb99580-123b-11ea-874e-9fcb95231ada.png">

These test cases are failing and all for same reason.